handbook/engineering/distribution/index.md

```diff
@@ -85,3 +85,19 @@ Go, Docker, Kubernetes
   * [How to simulate k8s admin security restrictions](k8s_admin_custom_policy.md)
   * [How to test the Gitlab native integration locally](gitlab_native_local.md)
   * [How to make updates to global settings and configuration on sourcegraph.com](update_sourcegraph_website.md)
+
+## Members
+
+- [G. P.](https://hire.lever.co/search/5f3b284f-04ce-4bdc-86ba-f9eed6a0ae9e) ([engieering manager](../roles.md#engineering-manager)) estimated start date in June.
+- [Stephen Gutekanst](../../../company/team/index.md#stephen-gutekanst) ([project lead](../roles.md#project-lead))
+- [Geoffrey Gilmore](../../../company/team/index.md#geoffrey-gilmore)
+- [Uwe Hoffmann](../../../company/team/index.md#uwe-hoffmann)
+- [Dave Try](../../../company/team/index.md#dave-try)
+- [Dax McDonald](../../../company/team/index.md#dax-mcdonald-he-him)
```

Comment:
thought: the pronouns make the anchor links a bit strange - maybe they could be converted to one of the dot point items instead (out of the header), so that the anchor links can just be the name?

Reply:
It's a shame GitHub Flavored Markdown doesn't have a way to override the auto-generated anchor. (And, annoyingly, the anchors rendered on GitHub are different to those rendered on about.sourcegraph.com: GitHub doesn't replace the slash with a hyphen.)

Against that, they should be stable once created, unless the bio writer did something silly like put in the wrong pronoun format. ([_cough_](https://github.com/sourcegraph/about/pull/1025)) Maybe we just accept the weirdness as a one time cost to keep them prominent?

Reply:
I think everyone owns their own entry on the team page and is free to decide if or where they want to indicate their pronouns. For example, [Quinn](https://about.sourcegraph.com/company/team#quinn-slack) indicated his pronouns as a bullet point and not in the header.